build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+import org.hypertrace.gradle.publishing.HypertracePublishExtension
+import org.hypertrace.gradle.publishing.License
+
+plugins {
+  id("org.hypertrace.repository-plugin") version "0.2.3"
+  id("org.hypertrace.ci-utils-plugin") version "0.1.4"
+  id("org.hypertrace.publish-plugin") version "0.3.3" apply false
+  id("org.hypertrace.jacoco-report-plugin") version "0.1.3" apply false
+}
+
+subprojects {
+  group = "org.hypertrace.core.kafkastreams.framework"
+  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
+    configure<HypertracePublishExtension> {
+      license.set(License.APACHE_2_0)
+    }
```

Comment:
you may also want to set source and target compatibility version to `JavaVersion.VERSION_11` here or in `kafka-streams-framework` build.gradle.kts

```
pluginManager.withPlugin("java") {
    configure<JavaPluginExtension> {
      sourceCompatibility = JavaVersion.VERSION_11
      targetCompatibility = JavaVersion.VERSION_11
    }
  }
```

Reply:
done